src/Symfony/Bundle/SecurityBundle/DependencyInjection/MainConfiguration.php

```diff
@@ -66,9 +66,11 @@ public function getConfigTreeBuilder()
                     return false;
                 })
                 ->then(function ($v) {
-                    $v['access_decision_manager'] = array(
-                        'strategy' => AccessDecisionManager::STRATEGY_AFFIRMATIVE,
-                    );
+                    if (!isset($v['access_decision_manager'])) {
+                        $v['access_decision_manager'] = array();
+                    }
```

Comment:
This part is not needed as PHP won't emit a notice when a directory is not defined and will happily create it on demand (line below).

Reply:
indeed, updated